planning/behavior_velocity_run_out_module/README.md

```diff
@@ -108,6 +108,14 @@ You can choose whether to use this feature by parameter of `use_partition_lanele
 
 ![brief](./docs/exclude_obstacles_by_partition.svg)
 
+##### Exclude obstacles that cross the ego vehicle's "cut line"
+
+This module can exclude can exclude obstacles that have predicted paths that come from behind the ego vehicle. It excludes obstacles if the their predicted path crosses the ego's "cut line". The "cut line" is generating a virtual line segment that is perpendicular to the ego vehicle and that passes through the ego's base link.
```

Comment:
```suggestion
This module can exclude obstacles that have predicted paths that come from behind the ego vehicle. It excludes obstacles if their predicted path crosses the ego's "cut line". The "cut line" is generating a virtual line segment that is perpendicular to the ego vehicle and that passes through the ego's base link.
```

Reply:
> This module can exclude obstacles that have predicted paths that come from BEHIND the ego vehicle.

the exclued targets are all objects that cross the cut line, not just the BEHIND?

Reply:
You are right, the explanation was not clear, it excludes obstacles if they pass the back part of the ego by crossing the line. I think it is better now, please confirm @kosuke55